tests/event/test_event_manager.py

```diff
@@ -0,0 +1,28 @@
+from hathor.event.storage.memory_storage import EventMemoryStorage
+from hathor.pubsub import HathorEvents
+from tests import unittest
+
+
+class BaseEventManagerTest(unittest.TestCase):
+    __test__ = False
+
+    def setUp(self):
+        super().setUp()
+        self.event_storage = EventMemoryStorage()
+        self.network = 'testnet'
+        self.manager = self.create_peer(self.network, event_storage=self.event_storage)
+        self.event_manager = self.manager.event_manager
+
+    def test_if_event_is_persisted(self):
+        self.manager.pubsub.publish(HathorEvents.NETWORK_BEST_BLOCK_FOUND,
+                                    event={"test": "test1"})
+        self.run_to_completion()
+        self.assertIsNotNone(self.event_storage.get_event(0))
+
```

Comment:
Should we have tests for some edge cases, like receiving 2 "group start events" in sequence, or 2 "group end events", or trying to save an event that has a non-sequential id?

Reply:
Yeah. I'll add tests for those.